FIRBoundaries.dat

```diff
@@ -9506,6 +9506,12 @@ LCCC-S|0|0|10|31.827319|30.000000|34.80502654|34.1982616995|33.40|32.00
 32.0104885036|33.9730497964
 31.827319|34.00055
 33.500000|30.000000
+LCCC-H|0|0|5|31.827319|30.000000|34.80502654|34.1982616995|33.52|30.54
+34.9123333|29.9993333
+34.0823333|32.5041667
+33.3716667|32.1797222
+32.8086111|31.6619444
+33.500000|30.000000

```

Comment:
This sector overlaps with `LCCC-S`. Is this intended? Also try keeping them ordered alphabetically, please.

Reply:
Yes the change is intended cause LCCC-H is a subsector of LCCC-S. Will move it alphabetically.

Reply:
Have some doubts about adding this split, in the stats database i cannot see more than 1 minute of use for LCCC-H, and no more than 6 minutes total for LCCC-C. it's not for me to decide, but is the extra 'clutter' really necessary? Is Cyprus going to use this split? 
![image](https://user-images.githubusercontent.com/42404718/96569675-b89dae00-12c9-11eb-8908-287cb16ebef0.png)